app/views/html_attachments/_print_meta_data.html.erb

```diff
@@ -0,0 +1,40 @@
+<br/>
+<br/>
+<br/>
+<p>
+<%= image_tag "open-government-licence.png", id: "meta-data-licence" %>
+</p>
+<br/>
+<br/>
+<br/>
+<p>
+  © Crown copyright <%= Date.today.year %>
```

Comment:
I don't know the answer to this, but should this be the year that the html attachment / it's edition was first published?  Or do we want to keep it constantly updated?